src/pages/modal.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from "react";
+
+interface ModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  children: React.ReactNode;
+}
+
+const Modal: FC<ModalProps> = ({ isOpen, onClose, children }) => {
```

Comment:
FC를 사용할 이유가 없는 것 같아요! 타입 붙이기가 필요하다면 이렇게 해주세요
```suggestion
const Modal = ({ isOpen, onClose, children }: ModalProps) => {
```